public/js/admin.js

```diff
@@ -47,6 +47,11 @@
     $("#messageDisplay").show().one();
   });
 
+  socket.on('expired token', function(service){
+    alert('Your session has expired. Please log in again.');
+    window.location.replace('https://cas.byu.edu/cas/logout');
+    window.location.replace('https://cas.byu.edu/cas/login?service=' + service);
```

Comment:
Ok so this just puts me in a forever loop. You might try having it redirect you to the logout and then the user would just go back a page or go back to hlrdesk.